_examples/gin_test.go

```diff
@@ -0,0 +1,24 @@
+package examples
+
+import (
+	"github.com/gavv/httpexpect"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestGinHandler(t *testing.T) {
```

Comment:
If we're adding a gin example, it would be reasonable to demonstrate something unique to gin.

One thing we could demonstrate is using httpexpext.NewBinder to test gin handler directly.

Other ideas?

Reply:
I'm not familiar  with the `NewBinder` method. You have a snippet to demonstrate?

Reply:
* https://github.com/gavv/httpexpect#use-http-handler-directly
* https://godoc.org/github.com/gavv/httpexpect#Binder

https://github.com/gavv/httpexpect/blob/0387206792097a6dc6fd830ac268a6f1953194f1/_examples/echo_test.go#L72-L87

Reply:
I've added the binder method for the test by following the echo example, the only issue I see with it is that it creates a new gin instance for each handler and doesn't feel very realistic, since you would normally use the same gin instance in every handler.

Reply:
I've modified the test to use `NewBinder` following the echo example. I did modify it a bit to pass along a `gin` instance to demonstrate that you would use the same instance in all of your route handlers and not create a new instance per route(hope that makes sense).